### PR TITLE
Created models associations

### PR DESCRIPTION
**Pull request by Bohdanvakh**

### CODE REVIEW

File: db/migrate/20230516153521_add_department_to_employees.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddDepartmentToEmployees < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :employees, :department, foreign_key: true
```
Comment (kvasyaa) [inline db/migrate/20230516153521_add_department_to_employees.rb]: try to always use `add_belongs_to`. It makes all job you need to do with `add_reference`

Comment (kvasyaa) [reply db/migrate/20230516153521_add_department_to_employees.rb]: `add_belongs_to :employees, :department`

**Review by kvasyaa — CHANGES_REQUESTED** [review]

File: db/migrate/20230516153521_add_department_to_employees.rb (hunk 1 above)
Comment (Bohdanvakh) [reply db/migrate/20230516153521_add_department_to_employees.rb]: what I should do with Department's Manager? Should it be a separate Manager model  or just a column in Department model?

**Review by kvasyaa — APPROVED** [review]

File: db/migrate/20230516153521_add_department_to_employees.rb (hunk 1 above)
Comment (kvasyaa) [reply db/migrate/20230516153521_add_department_to_employees.rb]: Let's skip it for now.

Comment (kvasyaa) [reply db/migrate/20230516153521_add_department_to_employees.rb]: and merge

**Pull request merged**